### PR TITLE
[#246] Use configured default resource for uploads

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

**Comment by trel** [comment]
this seems good upon inspection.   want to wait for feedback from peter?

**Comment by korydraughn** [comment]
i don't mind waiting on feedback from peter.

**Comment by trel** [comment]
ping @peterverraedt

File: src/metalnx-web/src/main/java/com/emc/metalnx/controller/UploadController.java [hunk 1]
```diff
@@ -110,6 +114,16 @@
 		String resources = multipartRequest.getParameter("resources");
 		String resourcesToUpload = multipartRequest.getParameter("resourcesToUpload");
 		String destPath = multipartRequest.getParameter("uploadDestinationPath");
+		
+		if ("undefined".equalsIgnoreCase(resourcesToUpload)) {
+		    // TODO Checking if the default resource is set should probably happen on
+		    // startup of the web application.
```
Comment (korydraughn) [inline src/metalnx-web/src/main/java/com/emc/metalnx/controller/UploadController.java:120]: did we want to remove this comment?

maybe we make it true in the future? that is, make the server report when a required option has not been defined.

Comment (trel) [reply src/metalnx-web/src/main/java/com/emc/metalnx/controller/UploadController.java:120]: yes, I'd say that's part of making #219 real

Comment (korydraughn) [reply src/metalnx-web/src/main/java/com/emc/metalnx/controller/UploadController.java]: removed comment.

**Review by peterverraedt — APPROVED** [review]
This should fix the issue

**Comment by korydraughn** [comment]
added #

**Pull request merged**